MovieViewer/MovieCell.swift

```diff
@@ -58,4 +60,13 @@ class MovieCell: UITableViewCell {
         customSelectionView.backgroundColor = themeColor
         selectedBackgroundView = customSelectionView
     }
+    
+    func toggleFavoriteStyle(flag: Bool) {
```

Comment:
this method a little confusing because it's not clear what `flag` means. can you have a more descriptive variable name here?